backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -4089,8 +4074,8 @@ root:
   inputDefinitions:
     parameters:
       text:
-        type: STRING
-schemaVersion: 2.0.0
+        parameterType: STRING
```

Comment:
This change is incomplete, the component spec above would have the same input parameter type definition.

Reply:
Thank you! Just updated.

Reply:
Sorry, I will add this back.